src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/VisitSessionsControllerTest.kt

```diff
@@ -125,9 +78,10 @@ class VisitSessionsControllerTest : IntegrationTestBase() {
       .exchange()
       .expectStatus().isOk
       .expectBody()
-      .jsonPath("$.length()").isEqualTo(1)
-      .jsonPath("$[0].startTimestamp").isEqualTo("2021-01-08T09:00:00")
-      .jsonPath("$[0].endTimestamp").isEqualTo("2021-01-08T10:00:00")
+      .jsonPath("$.length()").isEqualTo(3)
+      .jsonPath("$[0].endTimestamp").isEqualTo("2021-01-10T10:00:00")
```

Comment:
should this be 'startTimestamp'

Reply:
some of the tests need work,... I think the original was testing both start and end